src/specs/api.v2.yaml

```diff
@@ -1092,13 +1146,63 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/HTTPError"  
+                $ref: "#/components/schemas/HTTPError"
+    delete:
+      summary: Delete cell level metadata
+      description: Delete cell level metadata from an experiment
+      operationId: deleteCellLevelMeta
+      x-eov-operation-id: cellLevelMeta#deleteMetadata
```

Comment:
What about using names `cellLevelMeta#delete` and `cellLevelMeta#upload`, `cellLevelMeta#download`? I  don't want to be a pain, I have noticed that the code is much harder to read when there is no consistent naming scheme. Now is the time to define one. Can you choose one and rename everything accordingly?